selfdrive/car/hyundai/interface.py

```diff
@@ -82,6 +71,7 @@ def get_params(candidate, fingerprint, vin=""):
       ret.mass = 1985. + STD_CARGO_KG
       ret.wheelbase = 2.78
       ret.steerRatio = 14.4 * 1.1   # 10% higher at the center seems reasonable
+      tire_stiffness_factor = 0.444  # not optimized yet
```

Comment:
should be 1. Also, same for all cars, so we can use the default.

Reply:
optima and elantra use much lower values. should we keep these?

Reply:
yes, which you did already.